### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by pnwpedro**
### Description

Adds a PR template to the repo.

### Motivation and context

We'd like consistent PR descriptions.

### How was the change tested?

We'll see on the next PR!

### Change types

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (backwards-incompatible fix or feature)

### Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to Fauna documentation.
- [ ] My change requires a change to the README, and I have updated it accordingly.

### CODE REVIEW

**Review by cynicaljoy — APPROVED** [review]

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+### Description
+<!--- Describe your changes in detail -->
+
+### Motivation and Context
```
Comment (jrodewig) [inline .github/pull_request_template.md:6]: Nit: I'd use sentence case in headings. It tends to be easier to read.
```suggestion
### Motivation and context
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<!--- Provide a general summary of your changes in the Title above -->
```
Comment (jrodewig) [inline .github/pull_request_template.md:1]:
```suggestion
<!--- Provide a general summary of your changes in the title. -->
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+### Description
+<!--- Describe your changes in detail -->
```
Comment (jrodewig) [inline .github/pull_request_template.md:4]:
```suggestion
<!--- Describe your changes in detail. -->
```

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+### Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
```
Comment (jrodewig) [inline .github/pull_request_template.md:8]:
```suggestion
<!--- If it fixes an open issue, link to the issue. -->
```

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+### How Has This Been Tested?
```
Comment (jrodewig) [inline .github/pull_request_template.md:10]:
```suggestion
### How was the change tested?
```

File: .github/pull_request_template.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+<!--- Please describe in detail how you tested your changes. -->
```
Comment (jrodewig) [inline .github/pull_request_template.md:11]:
```suggestion
<!--- Describe how you tested your changes. -->
```

File: .github/pull_request_template.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+<!--- Include details of your testing environment, the tests you ran to -->
+<!--- see how your change affects other areas of the code, etc. -->
```
Comment (jrodewig) [inline .github/pull_request_template.md:13]:
```suggestion
<!--- Include details of your testing environment, tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->
```

File: .github/pull_request_template.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+### Screenshots (if appropriate):
+
+### Types of Changes
```
Comment (jrodewig) [inline .github/pull_request_template.md:17]:
```suggestion
### Change types
```

File: .github/pull_request_template.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change that fixes an issue)
+- [ ] New feature (non-breaking change that adds functionality)
+- [ ] Breaking change (a backwards incompatible fix or feature)
```
Comment (jrodewig) [inline .github/pull_request_template.md:21]:
```suggestion
- [ ] Breaking change (backwards-incompatible fix or feature)
```

File: .github/pull_request_template.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+
+### How Has This Been Tested?
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, the tests you ran to -->
+<!--- see how your change affects other areas of the code, etc. -->
+
+### Screenshots (if appropriate):
+
+### Types of Changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
```
Comment (jrodewig) [inline .github/pull_request_template.md:18]:
```suggestion
<!--- What types of changes does your code introduce? Put an `x` in any boxes that apply: -->
```

File: .github/pull_request_template.md [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+- [ ] Bug fix (non-breaking change that fixes an issue)
+- [ ] New feature (non-breaking change that adds functionality)
+- [ ] Breaking change (a backwards incompatible fix or feature)
+
+### Checklist:
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
```
Comment (jrodewig) [inline .github/pull_request_template.md:25]:
```suggestion
<!--- Review the following points. Put an `x` in any boxes that apply. -->
<!--- If you're unsure, don't hesitate to ask. We're here to help! -->
```

File: .github/pull_request_template.md [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+- [ ] My code follows the code style of this project.
+- [ ] My change requires a change to Fauna documentation.
+- [ ] My change requires a change to the README and I have updated it accordingly.
```
Comment (jrodewig) [inline .github/pull_request_template.md:28]:
```suggestion
- [ ] My change requires a change to the README, and I have updated it accordingly.
```

**Review by jrodewig — APPROVED** [review]
Looks great! I left some feedback — but it's all minor nits you can take or leave as you see fit. I'm comfortable with merging this as is.

**Pull request merged**